drivers/network/tcpip/lwip/src/Filelists.cmake

```diff
@@ -262,19 +262,19 @@ set(DOXYGEN_IN  ${LWIP_DIR}/doc/doxygen/lwip.Doxyfile.cmake.in)
 set(DOXYGEN_OUT ${LWIP_DIR}/doc/doxygen/lwip.Doxyfile)
 configure_file(${DOXYGEN_IN} ${DOXYGEN_OUT})
 
-find_package(Doxygen)
-if (DOXYGEN_FOUND)
-    message(STATUS "Doxygen build started")
-
-    add_custom_target(lwipdocs
-        COMMAND ${CMAKE_COMMAND} -E remove_directory ${DOXYGEN_DIR}/${DOXYGEN_OUTPUT_DIR}/html
-        COMMAND ${DOXYGEN_EXECUTABLE} ${DOXYGEN_OUT}
-        WORKING_DIRECTORY ${DOXYGEN_DIR}
-        COMMENT "Generating API documentation with Doxygen"
-        VERBATIM)
-else (DOXYGEN_FOUND)
-    message(STATUS "Doxygen needs to be installed to generate the doxygen documentation")
-endif (DOXYGEN_FOUND)
+#find_package(Doxygen)
+#if (DOXYGEN_FOUND)
+#    message(STATUS "Doxygen build started")
+#
+#    add_custom_target(lwipdocs
+#        COMMAND ${CMAKE_COMMAND} -E remove_directory ${DOXYGEN_DIR}/${DOXYGEN_OUTPUT_DIR}/html
+#        COMMAND ${DOXYGEN_EXECUTABLE} ${DOXYGEN_OUT}
+#        WORKING_DIRECTORY ${DOXYGEN_DIR}
+#        COMMENT "Generating API documentation with Doxygen"
+#        VERBATIM)
+#else (DOXYGEN_FOUND)
+#    message(STATUS "Doxygen needs to be installed to generate the doxygen documentation")
+#endif (DOXYGEN_FOUND)
```

Comment:
Well, this looks like a hack. And it must be breaking Doxygen in some way.

Reply:
I don't think that we have to maintain doxygen stuff for a third-party component?

Reply:
It might be important for the third-party components that are tightly integrated with ours.

Reply:
That should not need a target per library?
Which is what is being added here.

Reply:
> Well, this looks like a hack. And it must be breaking Doxygen in some way.

Yes, it's a "hack" in the sense that it modifies 3rd party code, which we do in many places. We don't need this here, so there is no reason to keep it.